### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by regineheberlein**
Do not merge until the workflow has been implemented

### CODE REVIEW

File: docs/aspace2alma/readme.md [hunk 1]
```diff
@@ -4,17 +4,25 @@ This workflow loads select collection-level archival records from ArchivesSpace
...
+Lib Jobs->>lib-sftp: aspace2alma requests barcode report at 3:30am daily; deletes report once downloaded
+Lib Jobs->>ASpace: aspace2alma requests MARC-XML for all collection-level ASpace records at 3:30am daily
 loop each Item
   Lib Jobs->>+Lib Jobs: applies Special Collections changes to default MARC-XML export
+  Lib Jobs->>+Lib Jobs: gets top_container records from ASpace
+  Lib Jobs->>+Lib Jobs: constructs item records from top_container records that 1.are at ReCAP 2. have a barcode 3.are not on the Alma barcode report
   Lib Jobs->>+Lib Jobs: adds select records to a single <marc:collection> wrapper
 end
-Lib Jobs->>lib-sftp: sends MARC-XML file to lib-sftp
+Lib Jobs->>lib-sftp: sends MARC-XML file to lib-sftp; renames old file
```
Comment (sandbergja) [inline docs/aspace2alma/readme.md:25]:
```suggestion
Lib Jobs->>lib-sftp: sends MARC-XML file to lib-sftp, renames old file
```

File: docs/aspace2alma/readme.md [hunk 2]
```diff
@@ -4,17 +4,25 @@ This workflow loads select collection-level archival records from ArchivesSpace
...
   Lib Jobs applies Special Collections changes to default MARC-XML export and adds select records to a single <marc:collection> wrapper.
-  Lib Jobs sends MARC-XML file to lib-sftp.
+  Lib Jobs gets top_container records from ASpace.
+  Lib Jobs constructs item records from top_container records that 1.are at ReCAP 2. have a barcode 3.are not on the Alma barcode report.
+  Lib Jobs sends MARC-XML file to lib-sftp; renames old file.
   ASpace to Alma imports profile loads MARC-XML file at 8am daily.
 }
-Lib Jobs->>ASpace: aspace2alma requests MARC-XML for all collection-level ASpace records at 4am daily
+Alma->>lib-sftp: Alma Analytics sends barcode report at 1am daily
+Lib Jobs->>lib-sftp: aspace2alma requests barcode report at 3:30am daily; deletes report once downloaded
```
Comment (sandbergja) [inline docs/aspace2alma/readme.md:17]:
```suggestion
Lib Jobs->>lib-sftp: aspace2alma requests barcode report at 3:30am daily, deletes report once downloaded
```

**Review by sandbergja — CHANGES_REQUESTED** [review]
Thanks, @regineheberlein !  It appears that mermaid doesn't like semicolons in the steps, so doesn't generate the chart.  Okay to change them to commas?

**Comment by regineheberlein** [comment]
Thanks @sandbergja

**Review by sandbergja — APPROVED** [review]
Thanks, @regineheberlein !

**Pull request merged**